app/models/review.rb

```diff
@@ -0,0 +1,10 @@
+class Review < ApplicationRecord
+  after_create :update_average_rating
+  belongs_to :restaurant
+
+  def update_average_rating
+    reviewsMetadata = restaurant.restaurant_reviews_metadatum
```

Comment:
Should be snake_case
Can be on the Restaurant model

Reply:
done